### ISSUE TITLE
HEAD dependency issues

### ISSUE THREAD

**Issue by pigfinite** [hansonrobotics/hrtool#6]
Hello,

I've been getting errors while trying to install HEAD via hrtool (and previously the legacy method). A while ago I tried installing HEAD via the old installation instructions but never had much luck with it since I had dependency issues. A similar thing seems to be happening here, and I think I'm missing a crucial step.
For example, when I get hrtool and run `hr install head` (on a brand new Ubuntu installation), I get a dpkg error:

> [INFO] Downloading head-deps-all_0.1.2_amd64.deb is done
Selecting previously unselected package head-deps-all.
(Reading database ... 204364 files and directories currently installed.)
Preparing to unpack .../head-deps-all_0.1.2_amd64.deb ...
Unpacking head-deps-all (0.1.2) ...
dpkg: dependency problems prevent configuration of head-deps-all:
 head-deps-all depends on blender; however:
  Package blender is not installed.
 head-deps-all depends on expect; however:
  Package expect is not installed.
 head-deps-all depends on festival; however:
  Package festival is not installed.
 head-deps-all depends on festival-dev; however:
  Package festival-dev is not installed.
 head-deps-all depends on gfortran; however:
  Package gfortran is not installed.
 head-deps-all depends on git; however:
  Package git is not installed.
 head-deps-all depends on libatlas-base-dev; however:
  Package libatlas-base-dev is not installed.
 head-deps-all depends on libblas-dev; however:
  Package libblas-dev is not installed.
 head-deps-all depends on liblapack-dev; however:
  Package liblapack-dev is not installed.
 head-deps-all depends on libopencv-dev; however:
  Package libopencv-dev is not installed.
 head-deps-all depends on openvpn; however:
  Package openvpn is not installed.
 head-deps-all depends on p
dpkg: error processing package head-deps-all (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 head-deps-all

The same error occurs when I run `hr install head-deps`.
I noticed there's an install command for OpenCog built-in to hrtool, so I tried that and it seemed to fix that problem, but then when trying to install HEAD again, it continued and then got errors about the HEAD packages being broken.

I guess what I'm mostly confused about is, does the installation process of HEAD require some manual management on the user's part at this point in time, or is the installation generally straightforward? On my machine it has taken a lot of work to get running, but I'm not experienced with system management by any means.

Thanks!

**Comment by pigfinite** [comment]
Not sure if this is the cause of my dependency issues, but adding my GitHub token to ~/.bashrc according to the README and testing the token doesn't print anything to the terminal. Also, running `hr install head-hr-ext` gives this error:
> [INFO] Installing head-hr-ext
jq: error: Cannot index string with string
jq: error: Cannot index string with string
jq: error: Cannot iterate over null
[ERROR] No asset is found for version 

I'm wondering if there's something wrong with the way my system is set up. Installing hrtool itself goes perfectly fine, but when I try to install the hrtool extension, I get errors.

**Comment by tesYolan** [comment]
It would pass that step with https://github.com/hansonrobotics/hrtool/blob/master/get_hr.bash#L3 changed to 0.8.12 (at least for me). @wenwei-dev Is there reason for that to be still at 0.8.6?

**Comment by pigfinite** [comment]
@tesYolan Thank you for pointing that out, that worked for me!
I don't know if I should open another issue for this, but after the installation I run `hr run robot` and I get an error about a setup.bash file being missing in the HEAD/devel directory in the workspace folder. However, I think hrtool didn't clone the source code anywhere, so I did it myself and built the code. Then running `hr run robot` started the system, but I got some errors, such as some Python modules not being found (like natsort and transitions, I think), and the WebUI didn't function correctly (blank webpage and some errors).

I'm still thinking that something might be wrong with my setup. I'm running Ubuntu 14.04.5 on a Macbook Pro. I'm wondering if there are any other steps I should take.

Thanks again
